docs/content/usages/events.md

```diff
@@ -4,37 +4,72 @@ sort: 6
 
 # Events API
 
-A Nikita session extends the [native Node.js Events API](https://nodejs.org/api/events.html). It provides a facility to listen to internal notification and know the current state of the program.
+A Nikita session extends the [native Node.js Events API](https://nodejs.org/api/events.html). It provides a facility to listen to internal notifications and know the current state of the program.
```

Comment:
This is wrong, we no longer extends the Events API.

Reply:
I replace with "use"?
`A Nikita session uses the native Node.js Events API...`

Reply:
Yes, but it is not Nikita, it is Nikita with the events plugin

Reply:
you agree with this?
`Nikita is plugged with the functionality which uses the native Node.js Events API. It provides a facility to listen to internal notifications and know the current state of the Nikita session.`